app/models/concerns/trackable.rb

```diff
@@ -0,0 +1,7 @@
+module Trackable
```

Comment:
add line in each model instead of making concern

Reply:
done